File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/Contratmariage.php
<?php

namespace MyApp\UserBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Contratmariage
 *
 * @ORM\Table(name="contratmariage", uniqueConstraints={@ORM\UniqueConstraint(name="UNIQ_A4262E4D65FBE7C7", columns={"numCinUtilisateur"})})
 * @ORM\Entity
 */
class Contratmariage
{
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_mariage", type="date", nullable=false)
     */
    private $dateMariage;

    /**
     * @var string
     *
     * @ORM\Column(name="id_epoux", type="string", length=10, nullable=true)
     */
    private $idEpoux;

    /**
     * @var string
     *
     * @ORM\Column(name="id_epouse", type="string", length=10, nullable=true)
     */
    private $idEpouse;

    /**
     * @var string
     *
     * @ORM\Column(name="validation", type="string", length=10, nullable=true)
     */
    private $validation;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_contrat", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idContrat;

    /**
     * @var \MyApp\UserBundle\Entity\Utilisateur
     *
     * @ORM\ManyToOne(targetEntity="MyApp\UserBundle\Entity\Utilisateur")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="numCinUtilisateur", referencedColumnName="id_uti")
     * })
     */
    private $numcinutilisateur;



    /**
     * Set dateMariage
     *
     * @param \DateTime $dateMariage
     * @return Contratmariage
     */
    public function setDateMariage($dateMariage)
    {
        $this->dateMariage = $dateMariage;

        return $this;
    }

    /**
     * Get dateMariage
     *
     * @return \DateTime 
     */
    public function getDateMariage()
    {
        return $this->dateMariage;
    }

    /**
     * Set idEpoux
     *
     * @param string $idEpoux
     * @return Contratmariage
     */
    public function setIdEpoux($idEpoux)
    {
        $this->idEpoux = $idEpoux;

        return $this;
    }

    /**
     * Get idEpoux
     *
     * @return string 
     */
    public function getIdEpoux()
    {
        return $this->idEpoux;
    }

    /**
     * Set idEpouse
     *
     * @param string $idEpouse
     * @return Contratmariage
     */
    public function setIdEpouse($idEpouse)
    {
        $this->idEpouse = $idEpouse;

        return $this;
    }

    /**
     * Get idEpouse
     *
     * @return string 
     */
    public function getIdEpouse()
    {
        return $this->idEpouse;
    }

    /**
     * Set validation
     *
     * @param string $validation
     * @return Contratmariage
     */
    public function setValidation($validation)
    {
        $this->validation = $validation;

        return $this;
    }

    /**
     * Get validation
     *
     * @return string 
     */
    public function getValidation()
    {
        return $this->validation;
    }

    /**
     * Get idContrat
     *
     * @return integer 
     */
    public function getIdContrat()
    {
        return $this->idContrat;
    }

    /**
     * Set numcinutilisateur
     *
     * @param \MyApp\UserBundle\Entity\Utilisateur $numcinutilisateur
     * @return Contratmariage
     */
    public function setNumcinutilisateur(\MyApp\UserBundle\Entity\Utilisateur $numcinutilisateur = null)
    {
        $this->numcinutilisateur = $numcinutilisateur;

        return $this;
    }

    /**
     * Get numcinutilisateur
     *
     * @return \MyApp\UserBundle\Entity\Utilisateur 
     */
    public function getNumcinutilisateur()
    {
        return $this->numcinutilisateur;
    }
}

File: Pidev_EGOV/Pi_Egov_Java/src/esprit/egov/entities/ContratMariage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package esprit.egov.entities;

import java.util.Date;


/**
 *
 * @author boumaiza
 */
public class ContratMariage {
    private Integer id_contrat;
    private Date date_mariage;
    private String id_epoux;
    private String id_epouse;
    private String validation;
    private Integer numCinUtilisateur;

    public ContratMariage() {
    }
    

    public ContratMariage(Integer id_contrat, Date date_mariage, String id_epoux, String id_epouse, String validation, Integer numCinUtilisateur) {
        this.id_contrat = id_contrat;
        this.date_mariage = date_mariage;
        this.id_epoux = id_epoux;
        this.id_epouse = id_epouse;
        this.validation = validation;
        this.numCinUtilisateur = numCinUtilisateur;
    }

    public ContratMariage(Date date_mariage, String id_epoux, String id_epouse, String validation, Integer numCinUtilisateur) {
        this.date_mariage = date_mariage;
        this.id_epoux = id_epoux;
        this.id_epouse = id_epouse;
        this.validation = validation;
        this.numCinUtilisateur = numCinUtilisateur;
    }

    public Integer getId_contrat() {
        return id_contrat;
    }

    public void setId_contrat(Integer id_contrat) {
        this.id_contrat = id_contrat;
    }

    public Date getDate_mariage() {
        return date_mariage;
    }

    public void setDate_mariage(Date date_mariage) {
        this.date_mariage = date_mariage;
    }

    public String getId_epoux() {
        return id_epoux;
    }

    public void setId_epoux(String id_epoux) {
        this.id_epoux = id_epoux;
    }

    public String getId_epouse() {
        return id_epouse;
    }

    public void setId_epouse(String id_epouse) {
        this.id_epouse = id_epouse;
    }

    public String getValidation() {
        return validation;
    }

    public void setValidation(String validation) {
        this.validation = validation;
    }

    public Integer getNumCinUtilisateur() {
        return numCinUtilisateur;
    }

    public void setNumCinUtilisateur(Integer numCinUtilisateur) {
        this.numCinUtilisateur = numCinUtilisateur;
    }
    
    
    
    
}
